src/App.jsx

```diff
@@ -1,35 +1,22 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import ProductDetail from './pages/ProductDetail';
+import Navbar from './components/Navbar';
+import Home from './pages/Home'
+import FetchingData from './components/FetchingData';
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
-}
+    <BrowserRouter>
+      <Navbar />
```

Comment:
Please take a look our [course module again for routing](https://neit.instructure.com/courses/56465/pages/live-coding-adding-react-router-to-week-2?module_item_id=2530831). This is not what we covered. We shouldn't be putting anything except for Routes within our `BrowserRouter`

In order to persist the toolbar correctly your routes would look something like this if you moved the routes to `main.jsx` like we did in class together.
```
<Routes>
   <Route path="/" element={<App />}>
          <Route index element={<Home />} />
          <Route path="effect-demo" element={<UseEffectDemo />} />
          <Route path="/fetchingData" element={<FetchingData/>} />
          <Route path="/productDetail/:id" element={<ProductDetail/>} />
    </Route>
</Routes>
```